src/components/ui/Chat/ChatBubble/index.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { HTMLAttributes } from 'react';
+
+import { BaseProps } from '../../Base';
+import { StyledChatBubble, StyledChatBubbleInfo, StyledChatBubbleContent } from './Styled';
+import PopOver from '../../PopOver';
+import PopOverItem, { PopOverItemProps } from '../../PopOver/PopOverItem';
+import { Dots } from '../../icons';
+
+export type MessageVariant = 'outgoing' | 'incoming';
+
+export interface ChatBubbleProps extends Omit<HTMLAttributes<HTMLDivElement>, 'css'>, BaseProps {
+    /* Determines styling for outgoing and incoming messages. */
+    variant: MessageVariant;
+    /* The name of the user that sent the message. */
+    senderName: string;
+    /* the unique identifier of the sender */
+    createdTimestamp: string;
+    /* The timestamp the message was last edited */
+    lastUpdatedTimestamp?: string;
+    /* The text content of the message. */
+    content: string;
+    /* The options available in the more options button. */
+    messageActions?: PopOverItemProps[] | null;
```

Comment:
Ok, updated. I'm not 100% in love with a prop called `messageActions` being type `ReactNode[]` (since it's an element, not a function) but I think you definitely right about moving away from the strict `PopOverItem[]` type.